sources/platform/collaboration/organization_account/index.md

```diff
@@ -60,8 +60,4 @@ For information on [adding members and assigning roles](./setup.md), see the Set
 
 Actor and task runs are billed to the account they are started from. **Always make sure you start your runs from the correct account** to avoid having an organization's runs billed to your personal account.
 
-The Free and Personal plans are [restricted to 9 seats](https://apify.com/pricing), while the Team plan comes with a minimum of 9, which you can increase later. If you are on the Team plan and want to add some more seats to the 9 it comes with, head over to your [Billing page and click on Subscriptions](https://console.apify.com/billing-new#/subscription). Click on **Upgrade**, and, in the **Add-ons** section, and add as many additional seats as you need. Then, just complete the upgrade process, and you're done.
```

Comment:
@gippy just to verify - it looks like we got rid of team seats, but I must have completely missed it. Just wanna make sure I'm not deleting important info